front/app/feed.xml/route.ts

```diff
@@ -13,23 +22,29 @@ export async function GET() {
     ttl: 6 * 60, // 6 hours
   });
 
-  const comics = await getAllMetadata();
-  const orderedComics = useComicMetadata(comics, "upload");
-  orderedComics.forEach((comic) => {
-    const firstImageLink = comic.series.comics[0].image_urls[0];
-    const firstImage = comic.images[firstImageLink];
-    const articleImageLink = getImageUrl(firstImage);
-    const description = firstImage.ocr || "A Comic";
+  const metadata = await getAllMetadata();
+  let comics : {comic: Comic, series: Metadata}[] = []
+  metadata.forEach((series) => {
+    Object.values(series.series.comics).forEach((comic) => {
+      comics.push({comic, series})
+    })
+  });
+  comics.sort(compare);
+  comics.length = 30;
+
+  comics.forEach((comic) => {
+    let description = ""
+    comic.comic.image_urls.forEach((img) => {
+      const image = comic.series.images[img]
+      description += '<img src="'+getImageUrl(image)+'" alt="'+image.ocr+'" width="'+image.width+'" height="'+image.height+'" />'
```

Comment:
```suggestion
      // Probably need to escape " in the ocr, 
      // so MoringMark does not do an injection attack here.
      // I swear they read my series extraction scripts and 
      // break them on purpose.
      description += `<img 
        src="${getImageUrl(image)}" 
        alt="${image.ocr}" 
        width="${image.width}" 
        height="${image.height}"
      />`
```

Let's use [string templating here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

We can also make it multiline for readability.

Reply:
yeah, that looks a lot better. made the change locally and will test